testnet/launcher/docker.go

```diff
@@ -210,49 +207,30 @@ func deployL1Contracts() (*node.NetworkConfig, error) {
 
 // waitForHealthyNode retries continuously for the node to respond to a healthcheck http request
 func waitForHealthyNode(port int) error { // todo: hook the cfg
-	requestURL := fmt.Sprintf("http://localhost:%d", port)
-	reqBody := `{"method": "obscuro_health", "id": 1}`
-
 	timeStart := time.Now()
 
+	hostRPCAddress := fmt.Sprintf("http://localhost:%d", port)
 	fmt.Println("Waiting for Obscuro node to be healthy...")
 	err := retry.Do(
 		func() error {
-			req, err := http.NewRequestWithContext(context.Background(), http.MethodGet, requestURL, bytes.NewBufferString(reqBody))
-			if err != nil {
-				return fmt.Errorf("client: could not create request: %w", err)
-			}
-			req.Header.Set("Content-Type", "application/json")
-
-			res, err := http.DefaultClient.Do(req)
+			client, err := rpc.NewNetworkClient(hostRPCAddress)
 			if err != nil {
 				return err
 			}
-			defer res.Body.Close()
+			defer client.Stop()
 
-			resBody, err := io.ReadAll(res.Body)
+			obsClient := obsclient.NewObsClient(client)
+			health, err := obsClient.Health()
 			if err != nil {
 				return err
 			}
-
-			response := map[string]interface{}{}
-			err = json.Unmarshal(resBody, &response)
-			if err != nil {
-				return err
+			if health {
+				fmt.Println("obscuro node is ready")
```

Comment:
Is this println here intentionally? 
If yes maybe capitalise Obscuro (`obscuro node is ready` -> `Obscuro node is ready`)

Reply:
```suggestion
				fmt.Println("Obscuro node is ready")
```